assignment08/js/script.js

```diff
@@ -0,0 +1,126 @@
+window.addEventListener('load', () => {
+    
+startClock();
+    //YOUR CODE SHOULD START BELOW THIS LINE
+
+navigator.geolocation.getCurrentPosition(youAreHere);
+    
+function youAreHere(position){
+   console.log(`Latitude: `+ position.coords.latitude `, Longitude: `+ 
+   position.coords.longitude +`.`);
+}
+
+const buttons = document.getElementsByTagName('button');
+
+for(const button of buttons){
+    const dataId = button.dataset.id;
+button.addEventListener('click', () => {
+    if(dataId == 'getCurrentWX'){
+        getCurrentWX();
+    }
+    else if(dataId == 'getFiveDay') {
+            getFiveDay();

```

Comment:
here is where having id in your html instead of dataID matters. First - you can't get the dataId before knowing which button triggered the event. Second - you need this function to track the event so that you can get which button was clicked:
button.addEventListener('click', (evt) => {
  if (evt.target.id == 'getCurrentWX') => { do stuff!

Make sense?